### ISSUE TITLE
[Question] Informations about codecs.

### ISSUE THREAD

**Issue by SergeantPL** [MaartenBaert/ssr#44]
Where do SSR looks for information about available codecs or does it have built-in list of codecs?
In list of codecs available in SSR there are missing few that I would like to test, especially VC3/DNxHD ( best for future editing ).
List of codecs from FFMpeg/AVConv: http://pastebin.com/TPAMgsHe
List of codecs from SSR: http://pastebin.com/yurP3BDh

Is there any way to add/enable them?

**Comment by morrolinux** [comment]
Hi, I see some time has passed and now dnxhd has been added to the dropdown codecs menu, but when pressing "record" it errors out with "unable to open codec". 
My ffmpeg installation has it enabled so I'm guessing it hasn't been implemented because of its 422 nature? 
In that case, have you got any suggestion for any other supported codec which is smooth enough for video editing?

**Comment by awensaunders** [comment]
Hi there, I'm also interested in dnxhd or prores screen recordings... I use Davinci as a video editor on linux, and they don't support h264/h265 in the free version, only more production-focused codecs like prores and dnx. My ffmpeg supports prores, but it doesn't show up in the list on simplescreenrecorder.

**Comment by HugoPoi** [comment]
SSR doesn't support `YUV422P` codecs, only `YUV420P`.

**Comment by MaartenBaert** [comment]
@HugoPoi You are mistaken, or using a really old version of SSR. SSR has supported YUV422 (and YUV444) for quite a while. That doesn't necessarily mean that every codec which uses YUV422 is supported by SSR.

**Comment by dromer** [comment]
Hmm, also ran into the "unable to open codec" error.

If the codec is not supported, then why have it in the dropdown?
